src/drt/src/db/drObj/drVia.h

```diff
@@ -319,9 +319,25 @@ class drVia : public drRef
   drBlockObject* owner_;
   FlexMazeIdx beginMazeIdx_;
   FlexMazeIdx endMazeIdx_;
-  bool tapered_ : 1;
-  bool bottomConnected_ : 1;
-  bool topConnected_ : 1;
+  bool tapered_;
+  bool bottomConnected_;
+  bool topConnected_;
```

Comment:
boost serialization cannot handle bitfields. if we want to have them as bitfields, they should be in a struct so that we could convert it to a readable datatype by boost.

Reply:
https://stackoverflow.com/questions/39008302/boostserialize-struct-with-bit-field-member-variables

Reply:
I could serialize them as strings if I want to I just didn't think it matters that much since we are not generally optimizing memory in tritonroute.

Reply:
The point of the link is you can do:

byte binary = tapered_;
ar & BOOST_SERIALIZATION_NVP(binary);
tapered_= binary;

which serializes the bitfield as a byte without requiring you to change it.  I wouldn't say we don't care about memory we just haven't focused on it.  drVia is not that important as it is local to a worker but it would matter on frVia or other more numerous objects.